assets/blocks/editor-components/toolbar-dropdown/index.js

```diff
@@ -24,62 +29,86 @@ import {
  * Dropdown for the editor toolbar.
  *
  * @param {Object}           props
- * @param {DropdownOption[]} props.options        Dropdown options.
- * @param {string}           [props.optionsLabel] Options label.
- * @param {Object}           props.icon           Icon for the toolbar.
- * @param {string}           props.value          Current dropdown value.
- * @param {Function}         props.onChange       Dropdown change callback, which receive
- *                                                the new value as argument.
+ * @param {DropdownOption[]} props.options          Dropdown options.
+ * @param {string}           [props.optionsLabel]   Options label.
+ * @param {Object}           props.icon             Icon for the toolbar.
+ * @param {string}           props.value            Current dropdown value.
+ * @param {Function}         props.onChange         Dropdown change callback, which receive the new value as argument.
+ * @param {Object}           props.toggleProps      Props passed to the toggle element.
+ * @param {Object}           props.popoverProps     Props passed to the popover component.
+ * @param {Function}         props.getMenuItemProps Render function for a menu item. Should return a props object.
  */
 const ToolbarDropdown = ( {
 	options,
 	optionsLabel,
 	icon,
 	value,
 	onChange,
+	toggleProps,
+	getMenuItemProps,
```

Comment:
We need to update this when calling the component too.

Reply:
Whoops, fixing in e79d36c